xml/art-obs-beginners-guide.xml

```diff
@@ -175,24 +167,28 @@ https://www.suse.com/communities/blog/suse-studio-integration/
    <title>Projects and Packages</title>
    <para>
     In &obsa;, packages are organized in <emphasis>projects</emphasis>.
-    A single project can contain several packages and serves a specific
-    purpose, defines access control, other repositories, and build targets
-    (operating systems and architectures).
+    A single project can contain several packages and usually serves a specific
+    organizational purpose. Generic access control, related repositories, and
+    build targets (operating systems and architectures) are all defined at
+    the project level.
    </para>
    <para>
     Projects can also contain other projects (subprojects) to structure work.
     Each project name is separated by colons.
```

Comment:
Ok, do we need to mention this exception about maintainers? 
I would try to avoid to explain another term and would leave this detail to the User Guide.

What about this?

```
Projects can also have other projects (subprojects) to structure work.
They are isolated from their parents projects and can be configured independently.
[...]
```

I'm not sure about the rest ("... can be configured independently.")